### DISCUSSION TITLE
Cannot copy link to highlighted code block via menu

### DISCUSSION THREAD

**Discussion by jsumners** [community/community#39246] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In the old code browser you could:

1. Click a line number
2. Hold shift and click a subsequent line number
3. Click the `...` menu next to the first line number that was clicked
4. Copy a permanent link to the highlighted code block

In the beta browser, click in the `...` menu removes the highlighted block and references only the first line clicked. A work around is to highlight the block, press `y`, and then copy the URL from the navigation bar.

https://user-images.githubusercontent.com/321201/202186256-ba215a21-cccb-4c89-b2ce-5f13672bbdf8.mp4

**Comment by david-engelmann** [comment]


**Reply by jsumners** [reply]
I do not understand how that is relevant to the issue described.

**Reply by david-engelmann** [reply]
I misunderstood the issue, I thought you werent able to copy the block.
